src/routes.js

```diff
@@ -108,6 +108,10 @@ function buildRoutes() {
     );
   });
 
+  if (routes.length === 0) {
+    console.error("No locale routes were constructed!");
```

Comment:
What was the purpose of this? Just curious.

Reply:
This was the root cause that made everything break on windows. Having this log is basically a log-o-matic regression test.